src/App.js

```diff
@@ -63,15 +63,19 @@ class App extends React.Component {
     )
 
     try {
-      this.loadUserAndCaseload()
```

Comment:
I would suggest this is not done in componentWillMount in general but noticed it is the standard way. Would also probably move the api calls to an async action/saga